back-end/api/quickstart/views.py

```diff
@@ -167,6 +167,31 @@ def retrieve(self, request, author, post):
         return Response(serializer.data)
 
 
+class FriendsListViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows for listing the friends of an author.
+    """
+    def list(self, request, author):
+        try:
+            author = Author.objects.get(id=author)
+            follow_queryset = Follow.objects.filter(receiver=author)
+            following_queryset = Following.objects.filter(sender=author)
+
+            friends = []
+            for follow in follow_queryset:
+                for following in following_queryset:
+                    if follow.sender["id"] == following.receiver["id"]:
+                        friends.append(follow.sender)
+
```

Comment:
`break`
in the case that there's accidentally multiple `Following` objects, the code right now would add the friend multiple times